content/2-2.md

```diff
@@ -159,33 +160,34 @@ Optional ideas to support assessment.
-* code functionality of fly-out menus so that it is usable by people using only the keyboard
-* mark up visible or hidden states of menu items using text in their labels or the WAI-ARIA attribute `aria-expanded`
+* code functionality of fly-out menus so that it is usable by people using different input devices including the keyboard only and voice interaction
+* mark up visible or hidden states of menu items using the WAI-ARIA attribute `aria-expanded`, and updating related labels accordingly
+* recite related requirements for designers to ensure that fly-out menus can be used effectively in different situations, such as using screen magnification and voice interaction
 
 #### Teaching Ideas for Topic
 
-Optional ideas to teach the learning outcomes.
+Optional ideas to teach the learning outcomes:
 
 * Show examples of menus that contain submenu items. Explain that the layer with the submenu items needs to be coded as a nested list, and that each of the submenu items needs to be coded as a list item. An example of the structure of menu and submenu items is provided in the WAI tutorials on [Indicate submenus](https://www.w3.org/WAI/tutorials/menus/flyout/#indicate-submenus).
-* Explain use of the WAI-ARIA attribute `aria-haspopup` to mark up the presence of submenu items. Indicate that this attribute should be present in the original HTML markup. An example of how to use the WAI-ARIA attribute `aria-haspopup` is provided in the WAI tutorials on [Indicate submenus](https://www.w3.org/WAI/tutorials/menus/flyout/#indicate-submenus).
-* Show examples of different ways to interact with fly-out menus, such as using the mouse or the keyboard. Examples of how to make functionality of fly-out menus available to all users are provided in the WAI tutorials on [Fly-Out functionality](https://www.w3.org/WAI/tutorials/menus/flyout/#fly-out-functionality). 
+* Explain the use of the WAI-ARIA attribute `aria-haspopup` to mark up the presence of submenu items. Indicate that this attribute should be present in the original HTML markup. An example of how to use the WAI-ARIA attribute `aria-haspopup` is provided in the WAI tutorials on [Indicate submenus](https://www.w3.org/WAI/tutorials/menus/flyout/#indicate-submenus).
+* Show examples of different ways to interact with fly-out menus, such as using the mouse, keyboard, or voice interaction. Examples of how to make functionality of fly-out menus available to all users are provided in the WAI tutorials on [Fly-Out functionality](https://www.w3.org/WAI/tutorials/menus/flyout/#fly-out-functionality). 
 * Explain that the WAI-ARIA attribute `aria-expanded` conveys if the submenu items are visible or hidden. Explain that the attribute needs to be present in the original HTML markup. Examples of how to use the `aria-expanded` attribute are provided in the WAI tutorials on [Indicate submenus](https://www.w3.org/WAI/tutorials/menus/flyout/#indicate-submenus).
```

Comment:
[Typo?] Should this be "Explain *what* the WAI-ARIA attribute..." rather than "Explain *that* the WAI-ARIA attribute..."?

Reply:
I used "that" as a shorthand for "the fact that" (the fact that it conveys if submenu items are visible or hidden) in an attempt for tersification. "Explain what the WAI-ARIA attribute `aria-expanded` conveys if the submenu items are visible or hidden" sounds weird to me. We could have just "Explain the WAI-ARIA attribute `aria-expanded` conveys", but this would be even more weird as it would miss the fundamental part of it, what does this attribute convey?.

Anyway, happy to change if this is better understood.